sound/soc/sof/ipc3-topology.c

```diff
@@ -2307,12 +2308,14 @@ static int sof_ipc3_tear_down_all_pipelines(struct snd_sof_dev *sdev, bool verif
 	 * topology loading the sound card unavailable to open PCMs.
 	 */
 	list_for_each_entry(swidget, &sdev->widget_list, list) {
-		if (swidget->dynamic_pipeline_widget)
+		if (swidget->dynamic_pipeline_widget) {
+			++dyn_widgets;
```

Comment:
do you need to count, a `bool` flag is not enough?

Reply:
@ujfalusi  Does that save anything?